libraries/app/database_api.cpp

```diff
@@ -1482,6 +1475,98 @@ map<string, committee_member_id_type> database_api_impl::lookup_committee_member
    return committee_members_by_account_name;
 }
 
+uint64_t database_api::get_committee_count()const
+{
+    return my->get_committee_count();
+}
+
+uint64_t database_api_impl::get_committee_count()const
+{
+    return _db.get_index_type<committee_member_index>().indices().size();
+}
+
+
+//////////////////////////////////////////////////////////////////////
+//                                                                  //
+// Workers                                                          //
+//                                                                  //
+//////////////////////////////////////////////////////////////////////
+
+vector<optional<worker_object>> database_api::get_workers(const vector<worker_id_type>& worker_ids)const
+{
+    return my->get_workers( worker_ids );
+}
+
+vector<optional<worker_object>> database_api_impl::get_workers(const vector<worker_id_type>& worker_ids)const
+{
+    vector<optional<worker_object>> result; result.reserve(worker_ids.size());
+    std::transform(worker_ids.begin(), worker_ids.end(), std::back_inserter(result),
+                   [this](worker_id_type id) -> optional<worker_object> {
+                       if(auto o = _db.find(id))
+                           return *o;
+                       return {};
+                   });
+    return result;
+}
+
+vector<optional<worker_object>> database_api::get_workers_by_account(account_id_type account)const
+{
+    return my->get_workers_by_account( account );
+}
+
+vector<optional<worker_object>> database_api_impl::get_workers_by_account(account_id_type account)const
+{
+    vector<optional<worker_object>> result;
+    const auto& workers_idx = _db.get_index_type<worker_index>().indices().get<by_account>();
+
+    for( const auto& w : workers_idx )
```

Comment:
Please avoid iterating through the whole index. Use `lower_bound` and check for end while iterating.